{"uuid": "0afeac8e-4ee2-43bf-a8bf-3e88dc4de8d1", "children": ["bbdbd0b4-01cd-4d73-85f3-ef12822c02ca"], "befores": [{"name": "feed_explorer_agent", "status": "broken", "statusDetails": {"message": "KeyError: 'llama-3.2'\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/OA/OA/tests/agent_trajectory/feed_explore.py\", line 13, in feed_explorer_agent\n    llm = get_available_providers()[model]\n          ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^\n"}, "start": 1730426938299, "stop": 1730426940519}], "start": 1730426938299, "stop": 1730426944601}